src/types.ts

```diff
@@ -75,8 +75,10 @@ export type AppSettingsAPIResponse<StreamChatGenerics extends ExtendableGenerics
     channel_configs: Record<
       string,
       {
+        reminders: boolean;
```

Comment:
i think this fails prettier: the properties should be in an alphabetical order

Reply:
Its a required property ... they get higher priority over optional ones